internal/controller/bindings/forwarder_controller.go

```diff
@@ -112,11 +120,112 @@ func (r *ForwarderReconciler) update(ctx context.Context, epb *bindingsv1alpha1.
 		"port", epb.Spec.Port,
 	)
 
+	// Get the KubernetesOperator
+
+	operators := ngrokv1alpha1.KubernetesOperatorList{}
+	if err := r.Client.List(ctx, &operators); err != nil {
+		log.Error(err, "failed to list KubernetesOperators")
+		return err
+	}
```

Comment:
Q. Why list the operators here rather than used the defined `managerOpts.releaseName` to find the associated `kind: KubernetesOperator`?

Reply:
I had started working on this before managerOpts.releaseName was included in the API manager. We don't have it plumbed through yet for the bindings-forwarder. I will make that change though.

Reply:
Fixed